services/user.py

```diff
@@ -0,0 +1,61 @@
+from db.models import User
+
+
+def create_user(
+        username: str,
+        password: str,
+        email: str = None,
+        first_name: str = None,
+        last_name: str = None,
+) -> User:
+
+    user = User.objects.create_user(
+        username=username,
+        password=password,
+    )
+
+    if email:
+        user.email = email
+
+    if first_name:
+        user.first_name = first_name
+
+    if last_name:
+        user.last_name = last_name
+
+    user.save()
+
+    return user
+
+
+def get_user(user_id: int) -> User:
+    return User.objects.get(id=user_id)

```

Comment:
Consider handling exceptions such as `User.DoesNotExist` when retrieving a user by ID. This will prevent the application from crashing if an invalid ID is provided.